$OpenBSD: patch-plugins_giggle-view-terminal_c,v 1.2 2011/05/22 17:02:35 jasper Exp $

Needed if we actually want a real shell inside the term

--- plugins/giggle-view-terminal.c.orig	Thu Apr 22 16:09:37 2010
+++ plugins/giggle-view-terminal.c	Thu Apr 22 16:09:57 2010
@@ -193,7 +193,7 @@ giggle_view_terminal_append_tab (GiggleViewTerminal *v
 	vte_terminal_fork_command (VTE_TERMINAL (terminal),
 				   shell ? shell : "/bin/sh",
 				   NULL, NULL, directory,
-				   FALSE, FALSE, FALSE);
+				   TRUE, TRUE, TRUE);
 
 	title = g_filename_display_name (directory);
 	label = view_terminal_create_label (view, terminal, title);
